engine/src/main/resources/assets/shaders/ssao_frag.glsl

```diff
@@ -68,7 +68,7 @@ void main() {
         // TODO: Holy... frustum ray and linearized depth - please!
         sampleDepth = reconstructViewPos(texture2D(texDepth, offset.xy).r * 2.0 - 1.0, gl_TexCoord[0].xy, invProjMatrix).z;
 
-        rangeCheck = smoothstep(0.0, 1.0, ssaoRad / abs(viewSpacePos.z - sampleDepth));
+        rangeCheck = smoothstep(0.0, 1.0, 0.25 * ssaoRad / abs(viewSpacePos.z - sampleDepth));
```

Comment:
Please no Magic Number. Assign that 0.25 value to a (final?) variable with a meaningful name. 

This gotta be the worst PR -EVER-: all lines within it have an issue! 
:rofl: